src/metabase/driver/crate.clj

```diff
@@ -0,0 +1,96 @@
+(ns metabase.driver.crate
+  (:require [metabase.driver :as driver]
+            [metabase.driver.generic-sql :as sql]
+            [clojure.set :as set]
+            (korma [core :as k])
+            (metabase.driver.crate [query-processor :as qp]
+                                   [util :as u]
+                                   [generic-sql :as gs]
+                                   [native :as n]))
+  (:import (clojure.lang Named)))
+
+(defn- column->base-type
+  "Map of Crate column types -> Field base types"
```

Comment:
for reference: Crate data types - https://crate.io/docs/reference/sql/data_types.html